lib/vsc/mympirun/option.py

```diff
@@ -94,7 +94,9 @@ def make_init(self):
                        "str", "store", None),
 
             "output-check-timeout": ("Set timeout threshold to print warning if no output has been produced yet (sec)",
```

Comment:
you should also clarify here that `output` means `stdout` (and/or `stderr`), not any other form of output (like result files).
maybe rephrase like `Warn when no stdout/stderr was seen after start (in seconds; negative number disables this test)`?